search/views/search_clients_engine_sources_daily.view.lkml

```diff
@@ -0,0 +1,58 @@
+include: "//looker-hub/search/views/search_clients_engines_sources_daily.view.lkml"
+
+view: +search_clients_engines_sources_daily {
```

Comment:
These measures I added deserve a close eye from the search team:
- Do these names make sense for what the query is doing?
- Are the descriptions correct?
- Do we need to remove some or add more variants?

Note closely what the UI gives you in the info field, which does include the SQL fields being summed (those will not, however, be available to see in visualizations):
<img width="200" alt="Screen Shot 2021-05-26 at 11 21 09 PM" src="https://user-images.githubusercontent.com/20819040/119761013-0fc7ba80-be79-11eb-9c0d-09e634ae4063.png">

Reply:
This looks good to me.